daemon/control.c

```diff
@@ -542,6 +548,58 @@ control_handle_cmd_list_guestos_configs(UNUSED const ControllerToDaemon *msg, in
 	mem_free0(results);
 }
 
+/**
+ * Handles list_guestos_configs cmd.
+ * Used in both priv and unpriv control handlers.
+ */
+static void
+control_handle_cmd_retrieve_logs(UNUSED const ControllerToDaemon *msg, int fd)
+{
+	control_message_t message = CONTROL_RESPONSE_CMD_FAILED;
+	char *data_logs_dir = "/data/logs/";
+	char *var_logs_dir = "/tmp/00000000-0000-0000-0000-000000000000/var/logs/";
```

Comment:
see comment below. I would prefer to let c0 decide were the logs should be placed. Another problem arise if c0 mounts a tmpfs in /var/logs as this would not be visible in the CML.